db/migrations/20220610144605_hmscontent_47.up.sql

```diff
@@ -0,0 +1,44 @@
+BEGIN;
+
+--
+-- repositories
+--
+CREATE TABLE IF NOT EXISTS repositories (
+    uuid UUID UNIQUE NOT NULL PRIMARY KEY,
+    created_at TIMESTAMP WITH TIME ZONE,
+    updated_at TIMESTAMP WITH TIME ZONE,
+    url VARCHAR(255) NOT NULL,
+    last_read_time TIMESTAMP WITH TIME ZONE DEFAULT NULL,
+    last_read_error VARCHAR(255) DEFAULT NULL,
+    refer_repo_config UUID DEFAULT NULL
+);
+
```

Comment:
we want a uniquness constraint on url i belive, since all repo_configs with the same url would share one repository entry

Reply:
thanks! updated